pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -555,17 +562,16 @@ func (p *PriorityQueue) getUnschedulablePodsWithMatchingAffinityTerm(pod *v1.Pod
 // but they are waiting for other pods to be removed from the node before they
 // can be actually scheduled.
 func (p *PriorityQueue) NominatedPodsForNode(nodeName string) []*v1.Pod {
-	p.lock.RLock()
-	defer p.lock.RUnlock()
-	return p.nominatedPods.podsForNode(nodeName)
+	// TODO: make podsForNode() public?
+	return p.PodNominator.(*NominatedPodMap).podsForNode(nodeName)
```

Comment:
Can we also add the `NominatedPodsForNode` to `PodNominator ` because it also the operation to maintain nominated Pods？

Reply:
Ignore me. I find you did it in the next pr https://github.com/kubernetes/kubernetes/pull/90415

Reply:
Exactly, that's in #90415 :)